stroeer/core/v1/article.proto

```diff
@@ -21,6 +21,7 @@ message Article {
   repeated Element elements = 10;
   repeated Author authors = 11;
   repeated string onwards = 12;
+  repeated Body body = 13;
```

Comment:
`bodies`?

Reply:
@thisismana @harryurban from a database perspective: wouldn't it be better if we have on `Body` message which could be persisted and loaded separately consisting of a list of `BodyNode` messages in the core package? WebArticlePage API should probably then have an algorithm to split this list into "pages".

Like this in core

```
message Article {
  Body body = 13;
}

message Body {
  repeated BodyNode children = 1;
}

message BodyNode {
}
```

Reply:
the way its persisted does not necessarily be equal to the way it shows up in the `Article`. I'd say optimize for the frontend.

So `Article` could hold a `repeaded BodyNode body` which indicates one element per virtual article page later on.

For database persistence reasons, we may chose to store something different that would allow easier read and write operations. Lets test if storing `bodies[]` or `BodyNode` works better?

Reply:
see https://github.com/stroeer/tapir/pull/43/commits/940442347f2f2056689b5b145781aa0b3ac3a182 for proposal @harryurban @bobaaaaa @thisismana